omero/developers/Tables.rst

```diff
@@ -301,6 +301,99 @@ Examples
    :source:`examples/OmeroTables/FindMeasurements.java <examples/OmeroTables/FindMeasurements.java>`
 
 
+Data display in OMERO.web
+^^^^^^^^^^^^^^^^^^^^^^^^^
+The following code creates a table for a dataset that contains a few images.
+
+.. code-block:: python
+
+  from getpass import getpass
+  from random import random
+  from omero.gateway import BlitzGateway
+  from omero.grid import LongColumn, StringColumn, DoubleColumn, ImageColumn
+  from omero.model import FileAnnotationI, OriginalFileI, ImageAnnotationLinkI, DatasetAnnotationLinkI, ImageI, DatasetI
+
+
+  def create_table(connection, dataset_id):
+      dataset = connection.getObject('Dataset', dataset_id)
+      images = [connection.getObject('Image', child.getId()) for child in dataset.listChildren()]
```

Comment:
This can just be `images = dataset.listChildren()`. That gives you a generator of ImageWrappers.
or if you expect to iterate through the images multiple times, then convert the generator to a list:
`images = list(dataset.listChildren())`